independent-publisher-connectors/Shopify Admin Store/apiProperties.json

```diff
@@ -0,0 +1,151 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "",
+          "scopes": [
+            "read_analytics",
+            "read_reports",
+            "write_orders",
+            "read_customers",
+            "write_customers",
+            "write_reports",
+            "read_content",
+            "write_content",
+            "read_product_listings",
+            "read_products",
+            "write_products",
+            "read_checkouts",
+            "write_checkouts",
+            "read_inventory",
+            "write_inventory",
+            "read_marketing_events",
+            "write_marketing_events",
+            "read_shopify_payments_disputes"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "authorizationUrlQueryStringTemplate": {
+              "value": "?client_id={ClientId}&response_type=code&redirect_uri={RedirectUrl}&response_mode=query&scope={Scopes}&prompt=login&state={State}"
+            },
+            "refreshBodyTemplate": {
+              "value": "client_id={ClientId}&refresh_token={RefreshToken}&redirect_uri={RedirectUrl}&grant_type=refresh_token&client_secret={ClientSecret}"
+            },
+            "scopeListDelimiter": {
+              "value": " "
+            },
+            "tokenBodyTemplate": {
+              "value": "client_id={ClientId}&code={Code}&redirect_uri={RedirectUrl}&grant_type=authorization_code&client_secret={ClientSecret}"
+            },
+            "authorizationUrlTemplate": {
+              "value": "{storeURL}/admin/oauth/authorize"
+            },
+            "tokenUrlTemplate": {
+              "value": "{storeURL}/admin/oauth/access_token"
+            },
+            "refreshUrlTemplate": {
+              "value": "{storeURL}/admin/oauth/access_token"
+            }
+          }
+        }
+      },
+      "storeURL": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify the url for your shop. e.g. storename.myshopify.com",
+          "displayName": "Shopify store URL - e.g. storename.myshopify.com",
+          "tooltip": "Provide your Shopify store URL"
+        }
+      },
+      "apiVersion": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify the API Version. eg. 2021-04",
+          "displayName": "Shopify API Version. eg. 2021-04",
+          "tooltip": "Provide your Shopify API Version"
+        }
+      }
+    },
+    "iconBrandColor": "#64943E",
```

Comment:
@thatapiguy Please refer the template for apiProperties.json file and change it to default iconBrandColor